package.json

```diff
@@ -28,6 +28,7 @@
     "react-firebase-hooks": "^2.1.0",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.2.0",
+    "storybook": "^5.3.0",
```

Comment:
Actually, do you mind reverting or dropping this commit? I opened #42 that upgrades all dependencies

Reply:
Let's get yours merged in first and  then I'll fix the conflict accordingly

Reply:
Sounds good 👍